src/cdk/v2/destinations/launchdarkly_audience/utils.js

```diff
@@ -0,0 +1,28 @@
+const { ACTION_TYPES, IDENTIFIER_KEY } = require('./config');
+
+/**
+ * Prepares a list of identifiers based on the provided data and identifier key.
+ * @param {*} listData The data containing lists of members to be added or removed from the audience.
+ * @returns
+ * {
+      "add": [
+          {
+           "id": "test@gmail.com"
+          }
+      ],
+      "remove": []
+   }
+ */
+const prepareIdentifiersList = (listData) => {
+  const list = {};
+  const processList = (actionData) =>
+    actionData
+      .filter((member) => member.hasOwnProperty(IDENTIFIER_KEY))
+      .map((member) => ({ id: member[IDENTIFIER_KEY] }));
```

Comment:
shouldn't `IDENTIFIER_KEY` be dynamic to know what `key` to pick from the payload ?

Reply:
Does rudder-sources handles this & always sends `identifier` in the payload even though warehouse table might not have it ?

Reply:
We are mapping user selected column to `identifier` in integration info repo. We will always recieve identifier in the payload to transformer.
https://github.com/rudderlabs/rudder-integrations-info/pull/163/files